### PR TITLE
Results updates - figures 4 and 5 (1 of 2)

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose
❗ staggered on #245 

Updating text for figure 5

### Issue

#204 
#203 

- Also updated commits to compiled figures currently updated in the analysis repo
- Added tmp figures 4 and 5

#### Which areas should receive a particularly close look?
- survival results: for the final results (medullo quantiseq), should I not include macrophage M1 significance results since the high CI is Inf? Apparently, there is another mathematical way to avoid Inf in small datasets, but I do not think we have time for that 😅 

#### Is there anything that you want to discuss further?
<s>- Note: for GSVA analyses, we are currently plotting and analyzing by broad histology, so that is how the results are written. It may make more sense to run this by cancer group (also questioned in this [PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1330#issue-1209685530)) and update the results. Right now, I kind of have to pull out cancers to discuss, but give the p-values of the broad histologies. But whether the cancer groups have high enough Ns for significance alone will remain to be seen.</s>
This has been updated to reflect cancer group

<s>- I added a new tmp figure 4, with a possible KM curve, so wrote up to include that.</s>

To assess p-values for GSVA, I am looking at this file:
```r
gs <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/analyses/gene-set-enrichment-analysis/results/gsva_tukey_stranded_cancer_group.tsv") %>%
  filter(significant_tukey_bonf == TRUE)
```

For this:
>Higher _TP53_ scores, indicative of dysregulation of the _TP53_ gene or pathway, while not significant across the entire OpenPBTA cohort, resulted in a significant overall survival risk for both DMGs (HR = 1.79e6, CI = 2-1.45e12, p = 0.04) and ependymomas (HR = 1612, CI = 9.1-3e5, p = 0.005).

I had to get this from a notebook since we did not do a FP for these cancer groups - should we just include this, or probably better, all fit results, into a supp table? This should probably be table 4, with 4 (software) being renamed to 5.

#### Is the pull request ready for review?
yes
### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by sjspielman** [comment]
> Note: for GSVA analyses, we are currently plotting and analyzing by broad histology, so that is how the results are written. It may make more sense to run this by cancer group (also questioned in this https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1330#issue-1209685530) and update the results. Right now, I kind of have to pull out cancers to discuss, but give the p-values of the broad histologies. But whether the cancer groups have high enough Ns for significance alone will remain to be seen.

Note that this has been done - https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1337

**Review by sjspielman — COMMENTED** [review]
A couple overall comments here - 

+ I don't think we should be adding more figure files to _this_ repository, or pointing to old commits. New compiled figures are going to be coming in next week to link directly from the analysis repository, so I would prefer either no stub, or holding off on this PR until those figures are in. If we leave in a stub or point to an old commit we risk missing replacing it later. It seems to me merging these old links in now just creates more work for us later.
+ That said, is there a reason old commits are being pointed to instead master? For example, Figure S1 is complete and doesn't need to be redone. I'd expect that link to point to the master branch in the analysis repo.

**Comment by jharenza** [comment]
> A couple overall comments here -
> 
> * I don't think we should be adding more figure files to _this_ repository, or pointing to old commits. New compiled figures are going to be coming in next week to link directly from the analysis repository, so I would prefer either no stub, or holding off on this PR until those figures are in. If we leave in a stub or point to an old commit we risk missing replacing it later. It seems to me merging these old links in now just creates more work for us later.

I didn't think we would want to merge the results without a figure, in case someone were reading and then get confused. I also didn't want that to hold up any copy-editing, so my preference would be to merge these remaining text PRs so you, @jaclyn-taroni and @jashapiro can get to editing, whether or not the links are finalized. I agree on holding off to update the commit links, so let's wait on all of those until the end?

> * That said, is there a reason old commits are being pointed to instead master? For example, Figure S1 is complete and doesn't need to be redone. I'd expect that link to point to the master branch in the analysis repo.

I think that was probably missed.

**Comment by sjspielman** [comment]
@jharenza just spoke with @jaclyn-taroni about whether to add these "temp" figures, and we agree it's a good idea to facilitate copy editing and sharing. I anticipate that compiled figures will be done by Wednesday of next week, and then maybe another day to get those linked merged up into the MS, so these temps seem like not a bad idea!!

File: content/07.supplemental.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+![**Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.** Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. Upset plot (F) showing overlap of variant calls. Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same tumor (N = 52 samples from 13 patients). Cumulative distribution TMB plots for PBTA (H) and TCGA (I) tumors using consensus SNV calls.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figureS2.png?sanitize=true){#fig:S2 tag="S2" width="7in"}
```
Comment (sjspielman) [inline content/07.supplemental.md:5]:
```suggestion
![**Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.** Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. Upset plot (F) showing overlap of variant calls. Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same tumor (N = 52 samples from 13 patients). Cumulative distribution TMB plots for PBTA (H) and TCGA (I) tumors using consensus SNV calls.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/master/figures/pngs/figureS2.png?sanitize=true){#fig:S2 tag="S2" width="7in"}
```

File: content/07.supplemental.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ## Supplemental Information Titles and Legends
 
-![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/489d729cde4318654a26b0a9ef0dee6d706352cd/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
+![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
```
Comment (sjspielman) [inline content/07.supplemental.md:3]:
```suggestion
![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/master/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
```

File: content/07.supplemental.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+![**Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.** (A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N < 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3). Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only primary tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. (B) Genome-wide plot of CNV alterations by broad histology. Each row represents one sample. Box and whisker plots of number of CNV breaks (C) or SV breaks (D) by number of chromothripsis regions.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figureS3.png?sanitize=true){#fig:S3 tag="S3" width="7in"}
```
Comment (sjspielman) [inline content/07.supplemental.md:7]:
```suggestion
![**Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.** (A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N < 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3). Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only primary tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. (B) Genome-wide plot of CNV alterations by broad histology. Each row represents one sample. Box and whisker plots of number of CNV breaks (C) or SV breaks (D) by number of chromothripsis regions.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/master/figures/pngs/figureS3.png?sanitize=true){#fig:S3 tag="S3" width="7in"}
```

File: content/07.supplemental.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 
-![**Related to Figure 3.** (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/489d729cde4318654a26b0a9ef0dee6d706352cd/figures/pngs/figureS4.png?sanitize=true){#fig:S4 tag="S4" width="7in"}
+![**Related to Figure 3.** (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figureS4.png?sanitize=true){#fig:S4 tag="S4" width="7in"}
```
Comment (sjspielman) [inline content/07.supplemental.md:9]:
```suggestion
![**Related to Figure 3.** (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/master/figures/pngs/figureS4.png?sanitize=true){#fig:S4 tag="S4" width="7in"}
```

File: content/07.supplemental.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+![**Related to Figure 4**. (A) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (B) and _TERC_ (C). First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (D), ependymoma (E), low-grade glioma (F), and high-grade diffuse astrocytic tumors (G). Box plots of quanTIseq estimates of immune cell proportions in histologies with more than one molecular subtypes with N >=3.](images/supplement/Figure-S5.png){#fig:S5 tag="S5" width="7in"}
```
Comment (sjspielman) [inline content/07.supplemental.md:11]: 👍

**Review by sjspielman — CHANGES_REQUESTED** [review]
Looks about good to go! I made some suggested changed to point to `master` branch for completed figures. It also needs a few words in the dictionary - 
```
proteogenomic
resect
immunologically
```

File: content/07.supplemental.md (hunk 2 above)
Comment (jharenza) [reply content/07.supplemental.md:3]: are you suggesting to untie the commits from these? There was a note in manubot formatting that the commit was required if this was from another repo, but I can try this if you want to go that route.

Comment (sjspielman) [reply content/07.supplemental.md:3]: Ah, I missed that info, can you point me to it about needing the specific commit? That documentation will help me understand why this has to be the case.

Comment (jharenza) [reply content/07.supplemental.md:3]: yes, no worries - I think @jaclyn-taroni was the one who initially reviewed when I had started adding the new figures once compiled: https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures

Comment (jharenza) [reply content/07.supplemental.md:3]: Note: although this appears that it pertains to only SVG, it was required that I added the commit/sanitize bits in order for the pngs to render as well.

File: content/03.results.md [hunk 6]
```diff
@@ -194,45 +193,79 @@ As expected, EXTEND scores significantly correlated with _TERC_ (R = 0.619, p <
...
+We used an additive model to perform multivariate cox regression on overall survival (**STAR Methods**), controlling for extent of tumor resection and whether a tumor was low-grade (LGG group) or high-grade (HGG group). 
+Depicted in the forest plot (**Figure {@fig:Fig4}F**), we show an expected significant overall survival benefit if the tumor was fully resected (HR = 0.36, CI = 0.2-0.67, p = 0.001) or if the tumor belongs to the LGG group (HR = 0.068, CI = 0.0091-0.5, p = 0.008).
+In contrast, we show an expected significant risk if the tumor belongs to the HGG group (HR = 5.7, CI = 3.5-9.0, p < 0.001).
+High telomerase scores were a poor prognostic indicator across brain tumor histologies (HR = 24, CI = 7.1-79, p < 0.001), demonstrating that EXTEND scores calculated from RNA-Seq can be used as a rapid surrogate measure of telomerase activity.
+Higher _TP53_ scores, indicative of dysregulation of the _TP53_ gene or pathway, while not significant across the entire OpenPBTA cohort, resulted in a significant overall survival risk for both DMGs (HR = 1.77e6, CI = 1.98-1.57e12, p = 0.04) and ependymomas (HR = 1612, CI = 9.1-2.9e5, p = 0.005).
```
Comment (jharenza) [inline content/03.results.md:232]: @sjspielman this result is currently only in a notebook - I am not sure if you saw my note about possibly making a supp table 4 for survival results mentioned in the paper and then moving KR as supp 5 - thoughts? I can do that, if so.

File: content/07.supplemental.md (hunk 2 above)
Comment (sjspielman) [reply content/07.supplemental.md:3]: Makes sense about provenance!

File: content/03.results.md (hunk 6 above)
Comment (sjspielman) [reply content/03.results.md:232]: This makes sense to me! Go for it.

**Review by sjspielman — APPROVED** [review]
Just noting that my review comments that commits ids in figure links should be changed to `master` are **correctly ignored.** Using specific commits preserves ["exact image provenance"](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures).

**Pull request merged**